scm-annotation-processor/src/main/java/sonia/scm/annotation/ScmAnnotationProcessor.java

```diff
@@ -112,7 +99,8 @@
     ImmutableSet.of(Subscribe.class.getName());
   private static final Set<ClassAnnotation> CLASS_ANNOTATIONS =
     ImmutableSet.of(new ClassAnnotation("rest-resource", Path.class),
-      new ClassAnnotation("rest-provider", Provider.class));
+      new ClassAnnotation("rest-provider", Provider.class),
+      new ClassAnnotation("mapper-modules", Mapper.class));
```

Comment:
Isn't this more a `mapper-implementation` or simply `mapper`?

Reply:
I renamed it yesterday like 3 times, because i thought `mapper` may not be specific enough. I will go with your suggestion.